src/hashes/identity.js

```diff
@@ -1,10 +1,14 @@
-// @ts-check
-
-import { from } from './hasher.js'
 import { coerce } from '../bytes.js'
+import * as Digest from './digest.js'
+
+export const code = 0x0
+export const name = 'identity'
+
+/**
+ * @param {Uint8Array} input
+ * @returns {Digest.Digest<typeof code, number>}
+ */
+export const digest = (input) => Digest.create(code, coerce(input))
 
-export const identity = from({
-  name: 'identity',
-  code: 0x0,
-  encode: (input) => coerce(input)
-})
+/** @type {import('./interface').SyncMultihashHasher<typeof code>} */
```

Comment:
How much does this matter here? Is it just for internal consistency _within_ this file? Is it showing up in the exported types?

Reply:
I should have added a code comment, this is mostly to ensure that module will end up compatible with desired interface as there is no way to say module `@implements {SyncMultihashHasher<typeof code>}`